{"name":"test_throws_sql_exception_when_add_User()","status":"passed","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1577966667482,"stop":1577966667482,"uuid":"4ea63c82-a13d-48dc-9cfb-d3612d49bf59","historyId":"45b95a25071531934bc72a2182270864","fullName":"com.webapp.service.database.dao.impl.UserDaoImplTest.test_throws_sql_exception_when_add_User","labels":[{"name":"host","value":"LAPTOP-072JRVQG"},{"name":"thread","value":"3532@LAPTOP-072JRVQG.main(1)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"com.webapp.service.database.dao.impl.UserDaoImplTest"},{"name":"testClass","value":"com.webapp.service.database.dao.impl.UserDaoImplTest"},{"name":"testMethod","value":"test_throws_sql_exception_when_add_User"},{"name":"suite","value":"com.webapp.service.database.dao.impl.UserDaoImplTest"}],"links":[]}